src/App.js

```diff
@@ -1,10 +1,43 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import LandingPage from './containers/LandingPage/LandingPage';
+import Home from './containers/Home/Home';
+import About from './containers/About/About';
+import Meals from './containers/Meals/Meals';
+import Workouts from './containers/Workouts/Workouts';
+import Blogs from './containers/Blogs/Blogs';
+import NotFound from './containers/NotFound/NotFound';
+import Navbar from './components/Navbar/Navbar';
+import Footer from './components/Footer/Footer';
+import RedirectToNotFound from './containers/NotFound/RedirectToNotFound';
+const DefaultRoutes = () => {
+  return (
+    <>
+      <Navbar />
+      <Switch>
+        <Route path="/home" component={Home} />
+        <Route path="/meals" component={Meals} />
+        <Route path="/workouts" component={Workouts} />
+        <Route path="/about" component={About} />
+        <Route path="/blogs" component={Blogs} />
+        <Route component={RedirectToNotFound} />
```

Comment:
replace this with this
```suggestion
        <Redirect to="notfound" />
```